src/app/constants/_colors.scss

```diff
@@ -1,3 +1,6 @@
 $wild-sand: #f4f4f4;
 $dusty-gray: #9b9b9b;
 $atlantis: #a5cd39;
+$alto: #d8d8d8;
+$tundora: #4a4a4a;
+$black: #000000;
```

Comment:
use shorthand
`#000`

Reply:
Can I just remove this variable? I'm using it to set the color of a text, which is black by default.

Reply:
👍